searchtool/server/views/layout.hbs

```diff
@@ -183,7 +183,8 @@
                           {{#if term}}
                             <button aria-expanded="false" aria-haspopup="true" data-toggle="dropdown" class="btn btn-default dropdown-toggle" id="btn-pgActions">Download <span class="caret"></span></button>
                             <ul aria-labelledby="btn-pgActions" class="dropdown-menu">
-                              <li role="presentation"><a target="_BLANK" href="/download?q={{term}}&dataset={{dataset}}&fromdate={{fromdate}}&todate={{todate}}&art_unit={{artunit}}&documentcode={{documentcode}}">Download CSV</a></li>
+                              <li role="presentation"><a target="_BLANK" href="/download?q={{term}}&dataset={{dataset}}&fromdate={{fromdate}}&todate={{todate}}&art_unit={{artunit}}&documentcode={{documentcode}}&textdata=yes">CSV with text</a></li>
```

Comment:
Nope!  The limit is in the call to Solr.  So regardless of how many rows there are, if they click on `with text`, it will limit the Solr response to 2k records, no more.  If they click on `without text`, it will limit the Solr response to 100k records, but of course, if there are less or more, it doesn't matter in this case.